docs/data/pulsar.yml

```diff
@@ -16,7 +16,7 @@
 # limitations under the License.
 ################################################################################
 
-version: 4.1.0
+version: 4.1-SNAPSHOT
```

Comment:
Ditto https://github.com/apache/flink-connector-pulsar/pull/67#discussion_r1422300397

Please keep it as 4.1.0

Reply:
```suggestion
version: 4.1.0
```